nucliadb_node/nucliadb_node/pull.py

```diff
@@ -109,6 +108,7 @@ async def closed_cb(self):
         logger.info("Connection is closed on NATS")
 
     async def initialize(self):
+        self.storage = await get_storage(service_name=SERVICE_NAME)
```

Comment:
IMO object attributes should be all defined in the constructor

Reply:
But it's an async func. We probably should add type definition for it at the top of the class though